org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/editor/validation/messages.properties

```diff
@@ -8,6 +8,7 @@ XtextValidatorConfigurationBlock_14=The grammar validation settings have changed
 XtextValidatorConfigurationBlock_15=Discouraged rule name:
 XtextValidatorConfigurationBlock_16=Info
 XtextValidatorConfigurationBlock_17=Usage of deprecated rules
```

Comment:
Could you please add a colon after 'Usage of deprecated rules', too?

Reply:
done :)